core/src/com/unciv/models/ruleset/Ruleset.kt

```diff
@@ -914,6 +915,8 @@ object RulesetCache : HashMap<String,Ruleset>() {
             this[ruleset.fullName] = Ruleset().apply {
                 load(fileHandle)
                 name = ruleset.fullName
+                modOptions.topics.add("unciv-mod")
+                modOptions.topics.add("unciv-mod-rulesets")
```

Comment:
I added these to the vanilla content so it displays them when selecting `rulesets`. I thought it might be cleaner than manually adding these to the list every time rulesets is selected

Reply:
>  I didn't even know the vanilla rulesets were currently shown in the mods menu

The do in the installed mods section. Tbh it might make more sense to just never show them, right? You can't delete/update them. You can look at them (lol) so since they're vanilla content it might be good to just never show them in the mods menu

Reply:
Was a consequence of the reordering I did when I split G&K and vanilla. Didn't really mind it at the time, though it is a bit weird